packages/java/neo-ms-auth-api/README.md

```diff
@@ -0,0 +1,92 @@
+# Neo Microservice Auth Api
+
+`neo-ms-auth-api` is a RESTful service developed in Java that provides JWT-based authentication functionalities. This service acts as an API for managing user authentication, leveraging the neo-auth-service crate to handle tokens and secure authentication.
+
+## Features
+
+- JWT-based Authentication: Generates and verifies JWT tokens for secure authentication.
+
+## Api documentation
+
+The API definition is available in the `swagger.yaml` file located in the root of the project. You can use Swagger Editor to visualize the API definition.
+
+## Installation
+
+To download the project, clone the repository from GitHub:
+
+```bash
+git clone https://github.com/xhermosilla/neocoder.git
+cd neocoder/packages/java/neo-ms-auth-api
+```
+
+## Usage
+
+### Prerequisites
+
+Make sure you have Java installed. 
+
+> If you don’t have it yet, you can install it from [here](https://www.oracle.com/es/java/technologies/downloads/).
+
+### Configuration
+
+The service requires some basic configurations, such as the secret key for tokens and other environment parameters. You can set these configurations on the `application.properties` file located at the src/main/resources folder:
```

Comment:
You can set these settings in the `application.properties` file located in the src/main/resources folder: